app/assets/stylesheets/layout/common/_header.scss

```diff
@@ -1,7 +1,7 @@
 .l-header {
-  background: $color-green-primary;
```

Comment:
Just moving the color to the inner element is problematic when you get to Desktop width, you no longer have a full-width header.